databases/week1/week1-homework.sql

```diff
@@ -0,0 +1,49 @@
+-- 1. Find out how many tasks are in the task table
+select count(*) 
+from task;
+
+-- 2.Find out how many tasks in the task table do not have a valid due date
+select count(*)
+from task
+where due_date IS NULL;
+
+-- 3.Find all the tasks that are marked as done
+select * from task 
+join status on task.status_id = status.id
+where status.name LIKE 'done%';
+
+-- 4. Find all the tasks that are not marked as done
+SELECT task.*,status.name
+FROM task INNER JOIN status ON task.user_id = status.id
```

Comment:
Hi, Sorry for the trouble, I have spotted an error here
Use the `task.status_id = status.id`
Thanks